### DISCUSSION TITLE
[Feature] Add support for MTS video files

### DISCUSSION THREAD

**Discussion by MMauro94** [immich-app/immich#2350] [Feature Request]
### The feature

I've tried uploading and `mts` video file. API fails with `400`, in the server logs:
```
[Nest] 1  - 04/28/2023, 8:35:26 PM   ERROR [AssetUploadConfig] Unsupported file type .mts file MIME type video/vnd.dlna.mpeg-tts
```
Since Immich is re-encoding the videos using ffmpeg anyway, I don't see a reason not to support this (and possibly many others file formats).

I've attached [mts.zip](https://github.com/immich-app/immich/files/11357161/mts.zip), that contains a a very short untouched mts clip.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by uhthomas** [comment]
`.MTS` is a weird kind of file. The mime type reported in your issue is `video/vnd.dlna.mpeg-tts`, but my browser correctly thinks it's `video/mp2t`. According to [Wikipedia](https://en.wikipedia.org/wiki/.m2ts), `.mts` and `.m2ts` are the same.

**Reply by MMauro94** [reply]
Is the mime type detected from the browser? I was using Chrome.

The file comes from a Panasonic Lumix GH4 camera btw.

**Reply by uhthomas** [reply]
I'm using Chromium on Linux. Not sure what's doing the mime type detection here.
